design/resources.go

```diff
@@ -94,6 +94,26 @@ var _ = Resource("workitemtype", func() {
 	})
 })
 
+var _ = Resource("trackers", func() {
+
+	BasePath("/trackers")
+
+	Action("create", func() {
+
+		Routing(
+			POST(""),
+		)
+		Description("Retrieve work item type with given id.")
```

Comment:
wrong description ? is this end point under trackers really returning workitems ?